### ISSUE TITLE
Failure creating a container using the tutorials example

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [kubernetes-incubator/cri-o#1387]
**Description**

To get a redis container inside of a Pod, the tutorial says to do the below but it errors.
```
# CONTAINER_ID=$(crictl create $POD_ID test/testdata/container_redis.json test/testdata/sandbox_config.json)
FATA[0000] Creating container failed: rpc error: code = Unknown desc = image not known
```
**Steps to reproduce the issue:**
1. Follow the tutorial's documentation.

**Describe the results you received:**
An error createng a container.

**Describe the results you expected:**
A container sans error.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `crio --version`:**

```# crio --version
crio version 1.9.0-dev
commit: "8ea79e755f5f89b132e642015a46d9d291b246ef"
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
VM running Fedora 27.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Can you figure out what is going on or fix the tuturial?

**Comment by TomSweeneyRedHat** [comment]
The tutorial was fixed in #1393.  Neglected to close this out after that merged, doing so now.
